plugins/query-insights/src/main/java/org/opensearch/plugin/insights/core/exporter/SinkType.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.insights.core.exporter;
+
+import java.util.Locale;
+
+/**
+ * Type of supported sinks
+ */
+public enum SinkType {
+    /** debug exporter */
+    DEBUG,
+    /** local index exporter */
+    LOCAL_INDEX;
```

Comment:
we can store the lower case string values against each to avoid toLowerCase on toString?